build-system/server/app.js

```diff
@@ -1560,7 +1560,7 @@ function generateInfo(filePath) {
 
 /**
  * @param {string} encryptedDocumentKey
- * @return {string|null}
+ * @return {?string}
```

Comment:
primitives themselves can't be nullable; need explicit syntax (how it is now)
```suggestion
 * @return {string|null}
```